src/data-table/measure-column-widths.js

```diff
@@ -62,7 +62,7 @@ type MeasureColumnWidthsPropsT = {
 };
 
 // sample size could likely be generated based on row count, to have higher confidence
-const MAX_SAMPLE_SIZE = 10;
+const MAX_SAMPLE_SIZE = 50;
```

Comment:
I changed this to handle the variable movie title lengths in a new example.. Signals to me that it needs to be configurable. Will create a new PR to do so